exonum_client/client.py

```diff
@@ -233,6 +233,42 @@ def available_services(self) -> requests.Response:
         """
         return _get(_SYSTEM_URL.format(self.schema, self.hostname, self.public_api_port, "services"))
 
+
+    def get_instance_id_by_name(self, name: str) -> int:
+        """
+        Gets an instance id with provided name.
+
+        Example:
+
+        >>> id = client.get_instance_id_by_name("cryptocurrency")
+        >>> id
+        42
+
+        Parameters
+        ----------
+        name: Name of the service.
+
+        Returns
+        -------
+        result: int
+            ID of the instance.
+        """
+        id = None
+        available_services = self.available_services().json()
```

Comment:
Calling "json()" without verifying the returned status code is a bit dangerous. It's not critical though, since it will mean an error either in exonum or in the light client and user won't be able to work-around the situation.